src/plugins/event-plugins/__tests__/JsErrorPlugin.test.ts

```diff
@@ -674,6 +719,69 @@ describe('JsErrorPlugin tests', () => {
         plugin.disable();
 
         // Assert
+        await waitForTick();
```

Comment:
Question: thanks for adding these. Were these causing test flakiness?

Reply:
This was needed because the tests were finishing before the record method has called resulting in failed tests. This is a result of changing recordHttpEventWithError() to an async method. This was the recommend way I found on jest to wait for async methods to finish. https://jestjs.io/docs/tutorial-async